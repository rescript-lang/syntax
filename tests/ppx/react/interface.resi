module A: {
  @react.component
  let make: (~x: string) => React.element
}

module NoProps: {
  @react.component
  let make: unit => React.element
}