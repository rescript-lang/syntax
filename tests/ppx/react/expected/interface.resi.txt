module A: {
  type props<'x> = {x: 'x}
  let make: React.componentLike<props<string>, React.element>
}

module NoProps: {
  type props = {}

  let make: React.componentLike<props, React.element>
}
